charts/spire/charts/tornjak-frontend/templates/deployment.yaml

```diff
@@ -56,8 +60,6 @@ spec:
               mountPath: {{ include "tornjak-frontend.workingDir" . }}/node_modules/.cache
             - name: env
               mountPath: {{ include "tornjak-frontend.workingDir" . }}/build/tmp
-            - name: logs
-              mountPath: /opt/app-root/src/.npm/
```

Comment:
Why remove logs?

Reply:
Giving the following error sometimes, putting it back since it's discovered to be a permission error. 
```
npm ERR! code ENOENT
npm ERR! syscall mkdir
npm ERR! path /home/node/.npm
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, mkdir '/home/node/.npm'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 
npm ERR! Log files were not written due to an error writing to the directory: /home/node/.npm/_logs
npm ERR! You can rerun the command with `--loglevel=verbose` to see the logs in your terminal
```

Reply:
I think there is maybe a slightly bigger problem here....

Some of the mounts are for layouts based on the openshift image that puts things in /opt/app-root/src/ and some of the mare for the vanilla images that put things in /home/node maybe. So one bug fix is not fixing it for the other image?

Reply:
I agree, thinking the samething.

Reply:
Resolved!